app/assets/templates/pages/home.html

```diff
@@ -59,76 +70,89 @@
   <div id="info-panel" class="container-fluid" ng-show="true">
     <div class="search-results" ng-show="search">
       <h1 ng-click="eee()">Search Results</h1>
-      <a href="" class="glyphicon glyphicon-chevron-right list-group-item" ng-click="switchPanel()"><div class="">Result 1</div></a>
-      <a href="" class="glyphicon glyphicon-chevron-right list-group-item" ng-click="switchPanel()"><div class="">Result 2</div></a>
-      <a href="" class="glyphicon glyphicon-chevron-right list-group-item" ng-click="switchPanel()"><div class="">Result 3</div></a>
-      <a href="" class="glyphicon glyphicon-chevron-right list-group-item" ng-click="switchPanel()"><div class="">Result 4</div></a>
-      <a href="" class="glyphicon glyphicon-chevron-right list-group-item" ng-click="switchPanel()"><div class="">Result 5</div></a>
-      <a href="" class="glyphicon glyphicon-chevron-right list-group-item" ng-click="switchPanel()"><div class="">Result 6</div></a>
-      <a href="" class="glyphicon glyphicon-chevron-right list-group-item" ng-click="switchPanel()"><div class="">Result 7</div></a>
+
+      <button class="list-group-item" ng-click="switchPanel()">
```

Comment:
Yes they are placeholders. Once we connect the data, then it would be
returning all the results (ng-repeat)



On Tue, Feb 24, 2015 at 1:26 PM, Harry Chen <notifications@github.com>
wrote:

> In app/assets/templates/pages/home.html
> <https://github.com/cchchoi1986/hikehk/pull/35#discussion_r25229635>:
>
> > @@ -59,76 +70,89 @@
> >    <div id="info-panel" class="container-fluid" ng-show="true">
> >      <div class="search-results" ng-show="search">
> >        <h1 ng-click="eee()">Search Results</h1>
> > -      <a href="" class="glyphicon glyphicon-chevron-right list-group-item" ng-click="switchPanel()"><div class="">Result 1</div></a>
> > -      <a href="" class="glyphicon glyphicon-chevron-right list-group-item" ng-click="switchPanel()"><div class="">Result 2</div></a>
> > -      <a href="" class="glyphicon glyphicon-chevron-right list-group-item" ng-click="switchPanel()"><div class="">Result 3</div></a>
> > -      <a href="" class="glyphicon glyphicon-chevron-right list-group-item" ng-click="switchPanel()"><div class="">Result 4</div></a>
> > -      <a href="" class="glyphicon glyphicon-chevron-right list-group-item" ng-click="switchPanel()"><div class="">Result 5</div></a>
> > -      <a href="" class="glyphicon glyphicon-chevron-right list-group-item" ng-click="switchPanel()"><div class="">Result 6</div></a>
> > -      <a href="" class="glyphicon glyphicon-chevron-right list-group-item" ng-click="switchPanel()"><div class="">Result 7</div></a>
> > +
> > +      <button class="list-group-item" ng-click="switchPanel()">
>
> Are these placeholders? Do we really need to commit these?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cchchoi1986/hikehk/pull/35/files#r25229635>.
>